addon-test-support/@ember/test-helpers/dom/fill-in.js

```diff
@@ -22,8 +22,8 @@ export default function fillIn(target, text) {
     if (!element) {
       throw new Error(`Element not found when calling \`fillIn('${target}')\`.`);
     }
-
-    if (!isFormControl(element) && !element.isContentEditable) {
+    let isControl = isFormControl(element);
```

Comment:
It turns out that `isContentEditable` is true for `input` elements in IE. I did this fix but maybe we have to go back to check if the element has a `contenteditable=true` attribute